model/general.yaml

```diff
@@ -1,4 +1,32 @@
 groups:
+  - id: general.client
```

Comment:
Why move it to general instead of having all in the original separate files?

Reply:
Yeah I think it would be best if we maintain the same separation, unless there's a reason to that I'm not aware?

Reply:
I was thinking, now that we move the attribute definitions into the registry, it would be a clearer structure if we group all the references / groups of attributes that are listed on https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attributes.md into a single file.

Reply:
@joaopgrassi what do you think about the above reasoning?

Reply:
We might need to split later both `docs/general` and `model/general`. Because it's not clear what general actually means, general to what? Especially with ongoing progress on adding security fields we might have different `general` fields for observability and security. Saying that I'm ok with having these fields all together right now and splitting later when need arrises

Reply:
I'm fine with the reasoning @AlexanderWert. If later we introduce security things, they can just go into `general/security.md/yaml`.